src/pages/catalog/oer/index.tsx

```diff
@@ -0,0 +1,244 @@
+          <p className="mb-1">
+            <strong>Shared by:</strong> {ownerName}{" "}
+            {ownerEmail && <a href={`mailto:${ownerEmail}`}>{ownerEmail}</a>}
+          </p>
+        )}
+        {institution && (
+          <p className="mb-1">
+            <strong>Institution: </strong> {institution}
+          </p>
+        )}
+        {github && (
+          <p className="mb-1">
+            <strong>View source</strong> on <Link href={github}>Github</Link>
+          </p>
+        )}
+      </div>
+    </article>
+  );
+};
+
+export default function Courses() {
+  const [showRequestCourseModal, setShowRequestCourseModal] =
+    React.useState(false);
+  return (
+    <React.Fragment>
+      <Head>
+        <title>OER | PrairieLearn</title>
+      </Head>
+
+      <PageBanner
+        title="Open Educational Resources (OER)"
+        subtitle="A catalog of questions to help you get started with PrairieLearn"
+      />
+
+      <div className="container-fluid my-3">
+        <div className="container-md">
+          <div className="alert alert-primary mb-0">
+            <p>
+              You&apos;ll be asked to sign in to PrairieLearn to view our
+              question catalog. If you want to use any of these questions, you
+              will first need to request your PrairieLearn course space, which
+              is always free for instructors.
+            </p>
+            <div className="col-md-12 text-center">
+              <button
+                className="btn btn-primary btn-lg"
+                onClick={() => setShowRequestCourseModal(true)}
+              >
+                Request a course
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div className="container-fluid my-5">
+        <div className="container-md">
+          <Heading>Publicly Shared Questions</Heading>
+          <p>
+            This question catalog has been publicly shared by instructors from
+            various universities. If you like any of these questions, you can
+            use them directly in your course, without any need to copy
+            questions. Read these{" "}
+            <Link href="https://prairielearn.readthedocs.io/en/latest/questionSharing/">
+              instructions
+            </Link>{" "}
+            to see how you can use publicly shared questions in your course.
+            Some of the questions can be copied to your course, so you can make
+            changes if you want!
+          </p>
+        </div>
+      </div>
+
+      <div className="container-fluid my-5">
+        <div className="container-md">
+          <div className={styles.grid}>
+            <SharedQuestionCard
+              image={eorphysics1}
+              title="Physics: Mechanics"
+              href="https://us.prairielearn.com/pl/public/course/2923/questions"
+              github="https://github.com/PrairieLearn/pl-oer-physics1"
+            ></SharedQuestionCard>
```

Comment:
Nit, you can use self-closing tags to make this a little easier to read.

```suggestion
            />
```

Reply:
fixed all instances as well